torch/_inductor/select_algorithm.py

```diff
@@ -1632,6 +1633,25 @@ def __call__(
         # TODO(nmacchioni): remove once CI tests are fixed
         choices = [choice for choice in choices if choice is not None]
 
+        if config.test_configs.autotune_choice_name_regex is not None:
+            choices = [
+                c
+                for c in choices
+                if re.search(
+                    config.test_configs.autotune_choice_name_regex,
```

Comment:
nit: it is it worth functools.lru_caching re.compile() and using that ?

Reply:
I imagine these regexes being used exclusively for testing (as in: not in prod / with real models) and being relatively simple (prob a substring or a few separated by |). So not sure how much it's worth precompiling them. Happy to add if you feel it's worth, though.